scheduler/config/config.go

```diff
@@ -299,6 +302,31 @@ type NetworkConfig struct {
 	EnableIPv6 bool `mapstructure:"enableIPv6" yaml:"enableIPv6"`
 }
 
+type NetworkTopologyConfig struct {
+	// Enable networkTopology services, including probe, network topology collection and synchronization service.
```

Comment:
`Enable network topology services, including probe, network topology collection and synchronization service.`